tests/test_pytest_picked.py

```diff
@@ -1,10 +1,14 @@
-from unittest.mock import patch
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
 import pytest
 
 
 def test_shows_affected_tests(testdir):
+    testdir.tmpdir.ensure_dir()
```

Comment:
this is two lines were added to ensure the tmpdir is actually a git repo, then git subprocess doesn't fail